app/messages.py

```diff
@@ -147,7 +148,7 @@
     "message": "A user with that " "email already exists."
 }
 USER_IS_NOT_REGISTERED_IN_THE_SYSTEM = {
```

Comment:
This name is a bit longer we can use `USER_NOT_REGISTERED` only here as you mentioned above. Remove that message and rename it. Also, it should be `"You are not registered in the system"`

Reply:
Ok I will change it

Reply:
> > @devkapilbansal I'm confused that I should remove line 25 & change line 150 is this the changes u want me to do?
> > But I change line 150 it is #Relation Constraints & that would hamper the message where ever it is previously initialized as USER_NOT_REGISTERED_IN_THE_SYSTEM.
> 
> Ohh, Yes you are right then. Just removed that extra message

ok